src/styles/main.scss

```diff
@@ -0,0 +1,52 @@
+.watch {
+  position: relative;
+  margin: 200px auto;
+  height: 400px;
+  width: 400px;
+  border: 1px solid deepskyblue;
+  border-radius: 100%;
+
+  &__arrow {
+    position: absolute;
+    bottom: 50%;
+    left: 49%;
+  }
+
+  &__first--sec {
+    transform-origin: bottom;
+    animation: move 60s steps(60) infinite normal;
+    height: 200px;
+    width: 5px;
+    border-radius: 10px;
+    background-color: blue;
+  }
+
+  &__first--min {
+    transform-origin: bottom left;
+    animation: move 3600s steps(60) infinite normal;
+    height: 150px;
+    width: 5px;
+    border-radius: 10px;
+    background-color: red;
+  }
+
+  &__point {
+    position: absolute;
+    bottom: 48%;
```

Comment:
Bad way of positioning. If you want to center something use 
bottom + transform translate

Reply:
ready